### PR TITLE
Feature/dfe 1013

### PR DESCRIPTION
**Pull request by negatron99**
Adding ability to sort the data on the Major axis by the default (the name / main column) or any of the data series added to the chart, either ascending or descending.

### CODE REVIEW

File: src/explore-education-statistics-admin/src/pages/prototypes/PrototypeChartTest.tsx.orig [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    axes: {
+      ...chartData.axes,
+
+      minor: {
+        ...chartData.axes.minor,
+        tickConfig: 'startEnd',
+        tickSpacing: '1',
+      },
+    },
+>>>>>>> origin/master
```
Comment (jamescutts) [inline src/explore-education-statistics-admin/src/pages/prototypes/PrototypeChartTest.tsx.orig]: Should this be in here?

File: src/explore-education-statistics-common/src/modules/find-statistics/components/charts/HorizontalBarBlock.tsx.orig [hunk 2]
```diff
@@ -0,0 +1,200 @@
+import {
+  ChartDataB,
+  ChartDefinition,
+  conditionallyAdd,
+<<<<<<< HEAD
```
Comment (jamescutts) [inline src/explore-education-statistics-common/src/modules/find-statistics/components/charts/HorizontalBarBlock.tsx.orig]: Should this be in here?

File: src/explore-education-statistics-common/src/modules/find-statistics/components/charts/LineChartBlock.tsx.orig [hunk 3]
```diff
@@ -0,0 +1,344 @@
+import {
+  ChartDataB,
+  ChartDefinition,
+  ChartProps,
+  conditionallyAdd,
+<<<<<<< HEAD
```
Comment (jamescutts) [inline src/explore-education-statistics-common/src/modules/find-statistics/components/charts/LineChartBlock.tsx.orig]: Should this be in here?

File: src/explore-education-statistics-admin/src/pages/prototypes/PrototypeChartTest.tsx.orig (hunk 1 above)
Comment (negatron99) [reply src/explore-education-statistics-admin/src/pages/prototypes/PrototypeChartTest.tsx.orig]: what the?

**Pull request merged**